tests/containers/search/filters/cfda/CFDAListContainer-test.jsx

```diff
@@ -0,0 +1,291 @@
+
+            const searchQuery = {
+                target: {
+                    value: '10'
+                }
+            };
+
+            // Call handleTextInput function
+            cfdaListContainer.instance().handleTextInput(searchQuery);
+
+            // Run fake timer for input delay
+            jest.useFakeTimers().runTimersToTime(1000);
+
+            await cfdaListContainer.instance().cfdaSearchRequest.promise;
+
+            // everything should be updated now
+            expect(handleTextInputSpy.callCount).toEqual(1);
+            expect(queryAutocompleteCFDASpy.calledWith(handleTextInputSpy));
+            expect(mockReduxActionCFDA).toHaveBeenCalledTimes(2);
+            expect(mockReduxActionCFDA.mock.calls[0]).toEqual([[]]);
+            expect(mockReduxActionCFDA.mock.calls[1]).toEqual([mockCFDA.results]);
+
+            // Reset spies
+            handleTextInputSpy.reset();
+            queryAutocompleteCFDASpy.reset();
+        });
+
+        it('should populate CFDA after performing the search', async () => {
+            // setup mock redux actions for handling search results
+            const mockReduxActionCFDA = jest.fn();
+
+            // setup the cfda list container and call the function to type a single letter
+            const cfdaListContainer = setup(Object.assign({}, initialFilters, {
+                setAutocompleteCFDA: mockReduxActionCFDA,
+                autocompleteCFDA: [],
+                selectedCFDA: new OrderedMap(),
+                selectCFDA: jest.fn()
+            }));
+
+            // Set up spies
+            const queryAutocompleteCFDASpy = sinon.spy(cfdaListContainer.instance(),
+                'queryAutocompleteCFDA');
+            const parseAutocompleteCFDASpy = sinon.spy(cfdaListContainer.instance(),
+                'parseAutocompleteCFDA');
+
+            cfdaListContainer.instance().queryAutocompleteCFDA('10');
+            await cfdaListContainer.instance().cfdaSearchRequest.promise;
+
+            // everything should be updated now
+            expect(queryAutocompleteCFDASpy.callCount).toEqual(1);
+            expect(parseAutocompleteCFDASpy.calledWith(queryAutocompleteCFDASpy));
+            expect(mockReduxActionCFDA).toHaveBeenCalledTimes(1);
+            expect(mockReduxActionCFDA.mock.calls[0]).toEqual([mockLocalCFDA]);
+
+            // Reset spies
+            queryAutocompleteCFDASpy.reset();
+            parseAutocompleteCFDASpy.reset();
+        });
+
+        it('should clear CFDA when the Autocomplete tells it to', () => {
+            const reduxState = [];
+
+            // setup mock redux actions for handling search results
+            const mockReduxAction = jest.fn((args) => {
+                expect(args).toEqual(reduxState);
+            });
+
+            const cfdaListContainer = setup(Object.assign({}, initialFilters, {
+                setAutocompleteCFDA: mockReduxAction,
+                autocompleteCFDA: [{
```

Comment:
👍 Noticed this wasn't the case in other filter tests, so I thought they needed to be locally kept.  Makes more sense though, updating accordingly.